cdap-docs/developers-manual/build.sh

```diff
@@ -86,6 +86,10 @@ function download_includes() {
 #   download_readme_file_and_test ${includes_dir} ${ingest_url} 277ded1924cb8d9b52a007f262820002 cdap-stream-clients/javascript
   download_readme_file_and_test ${includes_dir} ${ingest_url} b6dedd629c708dbc68bc918b768edda5 cdap-stream-clients/python
   download_readme_file_and_test ${includes_dir} ${ingest_url} 5fc88ec3a658062775403f5be30afbe9 cdap-stream-clients/ruby
+
+  echo_red_bold "Check included example files for changes"
+#   test_an_include 3b877d261bf74f7521a14369d186a21b ../../cdap-examples/WikipediaPipeline/src/main/scala/co/cask/cdap/examples/wikipedia/ScalaSparkLDA.scala
```

Comment:
I need to get the hashes of these files in before we can turn on the guards. I will do that in a later PR. Tracked with this issue: https://issues.cask.co/browse/CDAP-3133